src/loader.js

```diff
@@ -16,18 +17,14 @@ const reactModule = /['"]react['"]/;
  * @returns {string} The injected module source code.
  */
 function RefreshHotLoader(source, inputSourceMap) {
-  // Add dependency to allow caching and invalidations
-  this.addDependency(path.resolve('./runtime/RefreshModuleRuntime'));
-
   // Use callback to allow source maps to pass through
   this.callback(
     null,
     // Only apply transform if the source code contains a React import
     reactModule.test(source)
       ? source +
           '\n\n' +
-          Template.getFunctionContent(require('./runtime/RefreshModuleRuntime'))
-            .trim()
+          RefreshModuleRuntimeString.trim()
```

Comment:
Hi, I'm curious, Is it possible to move `trim()` to the variable declaration (Line 4)?

```suggestion
          RefreshModuleRuntimeString
```

Variable declaration:
```suggestion
const RefreshModuleRuntimeString = Template.getFunctionContent(RefreshModuleRuntime).trim();
```